modules/charon-core/src/main/java/org/wso2/charon3/core/exceptions/AbstractCharonException.java

```diff
@@ -25,7 +25,7 @@
  * implementers can use code property to decide which HTTP code needs to be set in header of the
  * response.
  */
-public class AbstractCharonException extends Exception {
+public class AbstractCharonException extends RuntimeException {
```

Comment:
shall we remove this change from this PR as it is already being tracked from https://github.com/wso2/charon/pull/122 ?

Reply:
would be great thx.

Reply:
failed to remove this change from this branch.... sorry